source/transactions/tests/unified/retryable-abort-handshake.yml

```diff
@@ -0,0 +1,88 @@
+description: "retryable abortTransaction on handshake errors"
+
+schemaVersion: "1.0"
```

Comment:
`serverless` syntax was introduced in schema version 1.4 (https://github.com/mongodb/specifications/commit/3586db8b1ed90d835737f6226bb3ff9012cfc5de).

Reply:
done!